source/simulator/assembly.cc

```diff
@@ -828,6 +831,8 @@ namespace aspect
             scratch.old_pressure);
         scratch.finite_element_values[introspection.extractors.pressure].get_function_values (old_old_solution,
             scratch.old_old_pressure);
+        scratch.finite_element_values[introspection.extractors.pressure].get_function_gradients (current_linearization_point,
+            scratch.current_pressure_gradients);
```

Comment:
We were hunting for bugs and found that this variable was not initialized in this function. But you are right, current_pressure_gradients is used nowhere in ASPECT. :-) I will create a PR.